public/app/features/alerting/unified/hooks/useFilteredRules.ts

```diff
@@ -109,6 +111,9 @@ const ufuzzy = new uFuzzy({
   intraSub: 1,
   intraTrn: 1,
   intraDel: 1,
+  // split search terms only on whitespace, this will significantly reduce the amount of regex permutations to test
+  // and is important for performance with large amount of rules and large needle
+  interSplit: '[w ]',
```

Comment:
probably? (untested)

```suggestion
  interSplit: '\\s+',
```

Reply:
Yep, this also works as intended!